web/index.html

```diff
@@ -3,8 +3,7 @@
 
 <head>
   <meta charset="UTF-8">
-  <link href='https://fonts.googleapis.com/css?family=Roboto:400,300,500' rel='stylesheet' type='text/css'>
-  <link href="https://fonts.googleapis.com/css?family=Ubuntu+Mono" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather" rel="stylesheet">
```

Comment:
Side note: there's a pretty nice open source font service made by a UBC alum (@alfredxing): https://github.com/alfredxing/brick 

Supposedly the fonts look nicer on it, I think it would be neat if we used this but let me know what you think.
https://brick.im/preview/brick.html
https://brick.im/preview/google.html

Reply:
Oh that is pretty neat! Can you see any difference though? My eyesight is pretty meh but I honestly can't tell 😛  Some other concerns:

- doesn't look like we can import all the weights and styles at once the way you can from google fonts
- potential downtime

Reply:
jk why not go UBC